blog/_posts/06-03-blogpost.md

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
I think the layout may get messed up with Jekyll if the very first line is left empty, I would move up the layout/title/date/author chunk up a line to be safe